src/components/ImageCard/ImageCard.jsx

```diff
@@ -13,7 +14,7 @@ import ImageHotspots from './ImageHotspots';
 const ContentWrapper = styled.div`
   height: 100%;
   max-height: 100%;
-  padding: 0 16px 16px 16px;
+  padding: 0 ${spacing05} ${spacing05} ${spacing05};
```

Comment:
can probably leave off the last padding value. I know this was present in the older code.
```suggestion
  padding: 0 ${spacing05} ${spacing05};
```